index.html

```diff
@@ -20,13 +20,13 @@
         <h1 class="name">Hey there. I’m<br>Mariane</h1>
         <h1 class="detail">I’m a software developer</h1>
         <p class="pOne">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-    </section>
-    <section id="socials">
-        <a href="#"><img src="./src/twitter.png" alt="twitter"></a>
-        <a href="#"><img src="./src/linkedin.png" alt="linkedin"></a>
-        <a class="mediumlogo" href="#"><img src="./src/medium.png" alt="medium"></a>
-        <a href="#"><img src="./src/github.png" alt="github"></a>
-        <a href="#"><img src="./src/angellist.png" alt="angellist"></a>
+        <div id="socials">
+        <a class="logos" href="#"><img src="./src/twitter.png" alt="twitter"></a>
+        <a class="logos" href="#"><img src="./src/linkedin.png" alt="linkedin"></a>
+        <a class="logos mediumlogo" href="#"><img src="./src/medium.png" alt="medium"></a>
+        <a class="logos" href="#"><img src="./src/github.png" alt="github"></a>
+        <a class="logos" href="#"><img src="./src/angellist.png" alt="angellist"></a>
```

Comment:
- [ ] It would be better if we change `<a class="logos">` to `<li class="logos">` to improve HTML semantics

Reply:
but  the li does not take the href="", So I'll put  an anchor <a> inside the <li class="logos">